eos_data_distribution/ndn/base.py

```diff
@@ -209,15 +211,28 @@ def registerPrefix(self, prefix=None, flags=None, *args, **kwargs):
         self._prefixes[prefix] = self._registerPrefix(prefix, flags, *args, **kwargs)
         return prefix
 
-    def _registerPrefix(self, prefix, flags=None,
+    def _registerPrefix(self, prefix, cost=None, controlParameters={},
```

Comment:
This will cause bugs. Don't use mutable values as default parameters, as you only get one object. Use `=None` and construct it inside.